packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -323,17 +351,44 @@ class SliderThemeData extends Diagnosticable {
   /// The color given to the [valueIndicatorShape] to draw itself with.
   final Color valueIndicatorColor;
 
-  /// The shape and behavior that will be used to draw the [Slider]'s thumb.
+  /// The shape that will be used to draw the [Slider]'s track.
+  ///
+  /// The [SliderTrackShape.getPreferredRect] method is used to to map
+  /// slider-relative gesture coordinates to the correct thumb position on the
+  /// track. It is also used to horizontally position tick marks, the slider is
+  /// discrete.
+  ///
+  /// The default value is [RectangularSliderTrackShape].
+  final SliderTrackShape trackShape;
+
+  /// The shape that will be used to draw the [Slider]'s ticks.
+  ///
+  /// The [SliderTickMarkShape.getPreferredSize] is used to help determine the
+  /// location of each tick mark on the track. It is also used as one of the
+  /// upper bounds for the min size of the slider.
+  ///
+  /// The default value is [RoundSliderTickMarkShape].
+  final SliderTickMarkShape tickMarkShape;
+
+  /// The shape that will be used to draw the [Slider]'s overlay.
+  ///
+  /// Both the [overlayColor] and a non default [overlayShape] may be specified.
+  /// In this case, the [overlayColor] is only used if the [overlayShape]
+  /// explicitly does so.
+  ///
+  /// The [SliderComponentShape.getPreferredSize] of this overlay shape, along
```

Comment:
You could probably put a single statement, roughly like this, in theme class doc. Instead of saying "used as one of the ...", as big as the largest of: ...

Reply:
Done